src/streamsync/core.py

```diff
@@ -45,6 +48,20 @@
     StreamsyncFileItem,
 )
 
+if TYPE_CHECKING:
+    from streamsync.app_runner import AppProcess
+
+
+def get_app_process() -> 'AppProcess':
+    from streamsync.app_runner import AppProcess  # Needed during runtime
+    raw_process: BaseProcess = \
```

Comment:
it's great to take that from the multiprocess directly I wasn't thinking of that.